doc/userguide/rules/payload-keywords.rst

```diff
@@ -552,9 +558,9 @@ Format::
 |		     | - dec - Converted string represented in decimal				|
 |		     | - oct - Converted string represented in octal				|
 +--------------------+--------------------------------------------------------------------------+
-| [dce]		     | Allow the DCE module determine the byte order 				|
+| [dce]              | Allow the DCE module to determine the byte order                         |
 +--------------------+--------------------------------------------------------------------------+
-| align <align-value>| Round the extracted value up to the next 				|
+| align <align-value>| Round the extracted value up to the next                                 |
 |                    | next <align-value> byte boundary post-multiplication (if any)            |
```

Comment:
Is this second `next` on purpose, or hiding in plain sight? :P

Reply:
Good catch .. thanks.